layers/core_checks/cc_render_pass.cpp

```diff
@@ -214,129 +214,128 @@ bool CoreChecks::ValidateDependencyCompatibility(const char *type1_string, const
+                                        const RENDER_PASS_STATE &rp2_state, const char *msg, const Location &loc,
                                         const char *error_code) const {
     const LogObjectList objlist(rp1_state.renderPass(), rp2_state.renderPass());
-    return LogError(objlist, error_code, "%s: RenderPasses incompatible between %s w/ %s and %s w/ %s: %s", caller, type1_string,
+    return LogError(error_code, objlist, loc, "RenderPasses incompatible between %s w/ %s and %s w/ %s: %s", type1_string,
                     FormatHandle(rp1_state).c_str(), type2_string, FormatHandle(rp2_state).c_str(), msg);
 }
 
 bool CoreChecks::LogInvalidDependencyMessage(const char *type1_string, const RENDER_PASS_STATE &rp1_state, const char *type2_string,
-                                             const RENDER_PASS_STATE &rp2_state, const char *msg, const char *caller,
+                                             const RENDER_PASS_STATE &rp2_state, const char *msg, const Location &loc,
                                              const char *error_code) const {
     const LogObjectList objlist(rp1_state.renderPass(), rp2_state.renderPass());
-    return LogError(objlist, error_code, "%s: RenderPasses incompatible between %s w/ %s and %s w/ %s: %s", caller, type1_string,
+    return LogError(error_code, objlist, loc, "RenderPasses incompatible between %s w/ %s and %s w/ %s: %s", type1_string,
                     FormatHandle(rp1_state).c_str(), type2_string, FormatHandle(rp2_state).c_str(), msg);
 }
 
 // Verify that given renderPass CreateInfo for primary and secondary command buffers are compatible.
 //  This function deals directly with the CreateInfo, there are overloaded versions below that can take the renderPass handle and
 //  will then feed into this function
 bool CoreChecks::ValidateRenderPassCompatibility(const char *type1_string, const RENDER_PASS_STATE &rp1_state,
-                                                 const char *type2_string, const RENDER_PASS_STATE &rp2_state, const char *caller,
-                                                 const char *error_code) const {
+                                                 const char *type2_string, const RENDER_PASS_STATE &rp2_state, const Location &loc,
+                                                 const char *vuid) const {
     bool skip = false;
 
     // createInfo flags must be identical for the renderpasses to be compatible.
     if (rp1_state.createInfo.flags != rp2_state.createInfo.flags) {
         const LogObjectList objlist(rp1_state.renderPass(), rp2_state.renderPass());
-        skip |= LogError(objlist, error_code,
-                         "%s: RenderPasses incompatible between %s w/ %s with flags of %u and %s w/ "
+        skip |= LogError(vuid, objlist, loc,
+                         "RenderPasses incompatible between %s w/ %s with flags of %u and %s w/ "
                          "%s with a flags of %u.",
```

Comment:
Let's burn those `%u`!
```suggestion
                         "RenderPasses incompatible between %s w/ %s with flags of %" PRIu32 " and %s w/ "
                         "%s with a flags of %" PRIu32 ".",
```